src/seraphis_mocks/mock_ledger_context.h

```diff
@@ -117,19 +119,22 @@ class MockLedgerContext final
         rct::keyV &proof_elements_out) const;
     /**
     * brief: max_legacy_enote_index - highest index of a legacy enote in the ledger
+    * param: ledger_indexing_amount -
     * return: highest legacy enote index (defaults to std::uint64_t::max if no enotes)
     */
-    std::uint64_t max_legacy_enote_index() const;
+    std::uint64_t max_legacy_enote_index(const rct::xmr_amount ledger_indexing_amount) const;
     /**
     * brief: max_sp_enote_index - highest index of a seraphis enote in the ledger
     * return: highest seraphis enote index (defaults to std::uint64_t::max if no enotes)
     */
     std::uint64_t max_sp_enote_index() const;
     /**
     * brief: num_legacy_enotes - number of legacy enotes in the ledger
+    * param: ledger_indexing_amount -
     * return: number of legacy enotes in the ledger
     */
-    std::uint64_t num_legacy_enotes() const { return max_legacy_enote_index() + 1; }
+    std::uint64_t num_legacy_enotes(const rct::xmr_amount ledger_indexing_amount) const
+    { return max_legacy_enote_index(ledger_indexing_amount) + 1; }
```

Comment:
```suggestion
    { return this->max_legacy_enote_index(ledger_indexing_amount) + 1; }
```
Always confusing seeing floating functions like this.